scripts/release.sh

```
@@ -0,0 +1,100 @@
+    # Fetch latest master info
+    echo "원격 master 브랜치 상태 확인 중..."
+    git fetch origin master --quiet
+
+    # Check for commits in master not in current branch
+    MISSING_COMMITS=$(git log HEAD..origin/master --oneline)
+    if [ -n "$MISSING_COMMITS" ]; then
+        echo "------------------------------------------------"
+        echo "⚠️  알림: 현재 브랜치에 반영되지 않은 master의 변경사항이 있습니다!"
+        echo "누락된 커밋 목록:"
+        echo "$MISSING_COMMITS" | head -n 5
+        [ $(echo "$MISSING_COMMITS" | wc -l) -gt 5 ] && echo "...외 $(($(echo "$MISSING_COMMITS" | wc -l) - 5))개의 커밋 더 있음"
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)